mitodl/bootcamp-ecommerce
# Feat: Allow users to specify bootcamp location in CMS

Adds a migration to change the AdmissionsSection's bootcamp_format and bootcamp_format_details to be bootcamp_location and bootcamp_location_details respectively.  Updates the UI where these values are displayed to label them as "Location" as well as displaying the value in the ApplicationDashboardPage.

#### What are the relevant tickets?
https://github.mit.edu/bootcamps/bootcamps-issues/issues/18

#### What's this PR do?
Allows user's to specify a Location and Location Details through the bootcamp CMS editor.

#### How should this be manually tested?

- Follow the steps in the "Where should the reviewer start?" section of this PR.
- visit the bootcamp course page (http://bc.odl.local:8099/bootcamps/collin-bootcamp/) which can be easily accessed by clicking the "Iive" link in the CMS.
- Apply for a program and view the application through http://bc.odl.local:8099/applications/
- Verify that both pages correctly show and label the location field and details.

#### Where should the reviewer start?

- Once you have bootcamp up and running locally, perform the initial step of seeding it with data (https://github.com/mitodl/bootcamp-ecommerce#seed-data).
- Perform a `docker-compose run web ./manage.py migrate` to ensure that you have the database changes.
- Go to <bootcamp_base_url>/cms and login with your admin user.
- Create a bootcamp index page, a bootcamp page, and finally an admissions page through CMS.  The pages don't need to be completely filled out or require any specific data.  On the admissions page ensure that you see the BOOTCAMP LOCATION and BOOTCAMP LOCATION DETAILS fields.  Enter data into those fields.
<img width="1326" alt="Screen Shot 2022-11-09 at 15 03 51" src="https://user-images.githubusercontent.com/8311573/200931149-09252a20-a2fe-4adc-8c12-519a3bcb0741.png">
<img width="1460" alt="Screen Shot 2022-11-09 at 15 03 41" src="https://user-images.githubusercontent.com/8311573/200931156-a72197ea-ad46-4a50-9519-f1c8a970f220.png">

#### Screenshots (if appropriate)
<img width="1466" alt="Screen Shot 2022-11-09 at 15 11 59" src="https://user-images.githubusercontent.com/8311573/200931427-c4dae2bc-6a35-4149-b3d2-e9cb32552e7f.png">
<img width="798" alt="Screen Shot 2022-11-09 at 15 12 13" src="https://user-images.githubusercontent.com/8311573/200931477-c54919cb-ce0c-4a88-9d8b-734799933972.png">
